src/attr/converters.py

```diff
@@ -139,8 +139,8 @@ def to_bool(val):
     """
     if isinstance(val, str):
         val = val.lower()
-    truthy = {True, "true", "t", "yes", "y", "on", "1", 1}
-    falsy = {False, "false", "f", "no", "n", "off", "0", 0}
+    truthy = {True, "true", "True", "t", "T", "yes", "Yes", "y", "Y", "on", "On", "1", 1}
```

Comment:
Looking at the code it looks like val should already be a lowercase string.  Am I reading this wrong?

Reply:
Yeah. The tests should pass without the changes IMO.